data/nasa_condensed.yaml

```diff
@@ -547,11 +547,14 @@ species:
   composition: {Br: 2}
   thermo:
     model: NASA7
-    temperature-ranges: [265.9, 332.503]
+    temperature-ranges: [265.9, 332.503, 6000.0]
     data:
     - [10.4252937, 0.111181227, -1.06856988e-03, 3.25976572e-06, -3.27490398e-09,
       -3506.20403, -49.0757083]
-    note: L 1/93
+    - [9.05669727, 0.0, 0.0, 0.0, 0.0, -2699.88017, -33.2936281]
+    note: >
+      L 1/93; This is a combination of the 7-coefficient polynomials below and above the
+      normal boiling point (332.503 K) given by McBride et al. (1993)
```

Comment:
Is this reference documented more formally somewhere (NTRS link would be fine, I think)?